eras/conway/impl/src/Cardano/Ledger/Conway/PParams.hs

```diff
@@ -572,6 +599,43 @@ conwayUpgradePParamsHKDPairs px pp =
   , ("dRepActivity", hkdMap px (toJSON @EpochNo) (pp ^. hkdDRepActivityL @era @f))
   ]
 
+instance ToJSON (UpgradeConwayPParams Identity) where
+  toJSON = object . upgradeConwayPParamsUpdatePairs
+  toEncoding = pairs . mconcat . upgradeConwayPParamsUpdatePairs
+
+upgradeConwayPParamsUpdatePairs :: KeyValue a => UpgradeConwayPParams Identity -> [a]
+upgradeConwayPParamsUpdatePairs upp =
+  uncurry (.=) <$> upgradeConwayPParamsHKDPairs upp
+
+upgradeConwayPParamsHKDPairs :: UpgradeConwayPParams Identity -> [(Key, Aeson.Value)]
+upgradeConwayPParamsHKDPairs UpgradeConwayPParams {..} =
+  [ ("poolThresholds", (toJSON @PoolVotingThresholds) ucppPoolVotingThresholds)
+  , ("dRepThresholds", (toJSON @DRepVotingThresholds) ucppDRepVotingThresholds)
+  , ("minCommitteSize", (toJSON @Natural) ucppMinCommitteeSize)
+  , ("committeeTermLimit", (toJSON @Natural) ucppCommitteeTermLimit)
+  , ("govExpiration", (toJSON @Natural) ucppGovActionExpiration)
+  , ("govDeposit", (toJSON @Coin) ucppGovActionDeposit)
+  , ("dRepDeposit", (toJSON @Coin) ucppDRepDeposit)
+  , ("dRepActivity", (toJSON @EpochNo) ucppDRepActivity)
+  ]
+
+instance FromJSON PoolVotingThresholds
+
+instance FromJSON DRepVotingThresholds
+
+instance FromJSON (UpgradeConwayPParams Identity) where
+  parseJSON =
+    withObject "UpgradeConwayPParams" $ \o ->
+      UpgradeConwayPParams
+        <$> o .: "poolThresholds"
+        <*> o .: "dRepThresholds"
+        <*> o .: "minCommitteeSize"
+        <*> o .: "cCTermLimit"
+        <*> o .: "govExpiration"
+        <*> o .: "govDeposit"
+        <*> o .: "dRepDeposit"
+        <*> o .: "dRepActivity"
```

Comment:
If we rename the fields in the serialization, we have to update these two. 
But at minimum we should fix the missing `e` in `minCommitteeSize`  which here in the deserialization is not missing